js/script.js

```diff
@@ -1,210 +1,38 @@
-	source: "Ralph Waldo Emerson"
-	},
-	{
-	quote: "In 20 years, you probably won't even remember this.",
-	source: "Anonymous"
-	},
-	{
-	quote: "Love all, trust a few, do wrong to none.",
-	source: "William Shakespeare"
-	},
-	{
-	quote: "Clear eyes, full hearts, can't lose.",
-	source: "Dillon Panthers",
-	citation: "Friday Night Lights",
-	year: "1990"
-	},
-	{
-	quote: "We are what we repeatedly do. Excellence, then, is not an act, but a habit.",
-	source: "Aristotle"
-	}
-];
-
-//Function to randomly select a quote value and return a random quote object from the quotes array
-function getRandomQuote () {
-	var randomNumber = Math.floor(Math.random() * (quotes.length));
-	var randomQuote = quotes[randomNumber];
-	return randomQuote;
-}	
-
-//Function to select random rgb color value
-function getRandomColor () {
-	var red = Math.floor(Math.random() * 256 );
-	var green = Math.floor(Math.random() * 256 );
-	var blue = Math.floor(Math.random() * 256 );
-	 var randomColor = 'rgb(' + red + ',' + green + ',' + blue + ')';
+function getRandomColor() {
+	var red = Math.floor(Math.random() * 256);
+	var green = Math.floor(Math.random() * 256);
+	var blue = Math.floor(Math.random() * 256);
+	var randomColor = 'rgb(' + red + ',' + green + ',' + blue + ')';
 	return randomColor;
 }
 
-//Function to call the getRandomQuote function and stores the returned quote object in a variable
-//Constructs a string containing the different properties of the quote object 
-function printQuote () {
-		var quotes = getRandomQuote ();
-		var quoteContainer = document.getElementById("quote-box");
-		var quoteString = `<p class="quote">${quotes.quote}</p><p class="source">${quotes.source}`;
-				if (quotes.citation) {quoteString += `<span class="citation">${quotes.citation}</span>`}
-				if (quotes.year) {quoteString += `<span class="year">${quotes.year}</span></p>`}
-				else {quoteString += '</p>'};
-					quoteContainer.innerHTML = quoteString;
-
-					//assigns random color value to document background color 
-					document.body.style.backgroundColor = getRandomColor ();
-}
-
-//Quote automatically refreshes every 15 seconds
-window.setInterval(function(){
-printQuote ();
-}, 15000);
-
-//Event listener on LoadQuote button to generate new quote		
-document.getElementById("loadQuote").addEventListener("click", printQuote, false);
-
-
-
+document.addEventListener("DOMContentLoaded", () => {
+	// DOM elements
+	const button = document.querySelector("button");
+	const quote = document.getElementById("quote");
+	const cite = document.getElementById("author");
+
+	async function updateQuote() {
+		// Fetch a random quote from the Quotable API
+		const response = await fetch("https://api.quotable.io/random");
+		const data = await response.json();
+		if (response.ok) {
+			// Update DOM elements
+			quote.textContent = data.content;
+			cite.textContent = data.author;
+		} else {
+			quote.textContent = "An error occured";
```

Comment:
I think it's better if another element is used to display an error message.

Alert might also do the job. And we leave `quote.textContent` the same as it was.